docs/Add-a-module.md

```diff
@@ -0,0 +1,63 @@
+
+## **How to add a module**
+
+
+1. Get the Module From this website [http://www.azerothcore.org/modules-catalogue/](http://www.azerothcore.org/modules-catalogue/)
+2. Clone it, or download.
+3. I'll use this for Exmple: [http://www.azerothcore.org/modules-catalogue/details.html?id=177019524](http://www.azerothcore.org/modules-catalogue/details.html?id=177019524).
+4. Once you downloaded it or cloned: ( Extract it, you will have than Folder looks like this :
+[[images/1.png]]
+
+
+5. Copy this folder you will have to paste it in next steps:
+[[images/2.png]]
+
+6. Now go in your AzerothCore folder in my case it's: C:\azerothcore-wotlk-master
```

Comment:
Please reword in: Now go in your AzerothCore folder (for example `C:\azerothcore-wotlk-master`)